ixmp/backend/base.py

```diff
@@ -81,6 +81,46 @@ def get_nodes(self):
         set_node
         """
 
+    @abstractmethod
+    def get_timesteps(self):
```

Comment:
suggest to rename to `timeslice` rather than `timestep` for generality

Reply:
renamed